apps/OpenSpace/ext/launcher/src/profile/horizonsdialog.cpp

```diff
@@ -296,6 +295,7 @@ void HorizonsDialog::createWidgets() {
         _startEdit = new QDateTimeEdit;
         _startEdit->setDisplayFormat("yyyy-MM-dd  T  hh:mm:ss");
         _startEdit->setDate(QDate::currentDate().addYears(-1));
+        _startEdit->setAccessibleName("Set start time");
```

Comment:
When trying with the NVDA screenreader it will read something like "set start time, 2023-01-09 T 00:00:00" and since the format is fixed maybe it's fine? Tabbing through the input field will go year -> month -> day -> time and at each point, it will do a readout of the value. 

but I guess adding a description like so `setAccessibleDescription("time format: YYYY-MM-DD T HH:MM:SS") isn't a bad idea?

Reply:
If the format is read, I guess that's fine. Was mostly a thought. 

And what I meant was rather not having it spell out the entire format, but maybe mention that its's given as a ISO 8601-like date string, in UTC. That is probably information that everyone could benefit from, however, so might be worth adding that information in some other way :)